src/buckets/agent/liccache.c

```diff
@@ -257,8 +257,8 @@ FUNCTION long get_rfpk(PGconn *pgConn, cacheroot_t *pcroot, char *rf_shortname)
 FUNCTION long add2license_ref(PGconn *pgConn, char *licenseName)
 {
     PGresult *result;
-    char  query[256];
-    char  insert[256];
+    char  query[512];
+    char  insert[512];
```

Comment:
Will it be a good idea to use `MAXSQL` like https://github.com/fossology/fossology/blob/bd139437c967d88313d7259915be103d25d3712d/src/delagent/agent/util.c#L111 ?

Also, please change following `sprintf`s to `snprintf`s accordingly.

Reply:
Yes.